.github/workflows/au4_build_macos.yml

```diff
@@ -86,6 +86,15 @@ jobs:
         echo "SENTRY_URL=$SENTRY_URL" >> $GITHUB_ENV
         echo "SENTRY_URL: $SENTRY_URL"
 
+    - name: Install Qt
+      uses: jurplel/install-qt-action@v4
+      with:
+        version: '6.9.2'
+        host: 'mac'
+        target: 'desktop'
+        arch: 'clang_64'
+        modules: 'qt5compat qtnetworkauth qtscxml qtshadertools qtwebsockets'
```

Comment:
You might want to check if `qtscxml` is necessary for Audacity. In MU, it's only needed because one legacy widget uses Qt StateMachine, and therefore I doubt Audacity needs it.